libs/utils/decorate.js

```diff
@@ -6,7 +6,8 @@ export function decorateButtons(el, size) {
   const buttonTypeMap = { STRONG: 'blue', EM: 'outline', A: 'blue' };
   buttons.forEach((button) => {
     const parent = button.parentElement;
-    const buttonType = buttonTypeMap[parent.nodeName] || 'outline';
+    let buttonType = buttonTypeMap[parent.nodeName] || 'outline';
+    if (parent.nodeName === 'EM' && parent.parentElement.nodeName === 'STRONG') buttonType = 'fill';
```

Comment:
I think we'll have to check for the reverse of this too.

Reply:
I'm also noticing all <em> references are completely removed from the DOM when nested with a <strong>

Reply:
@rgclayton Thanks for the finds on that. I have updated my PR w/ some adjustments.
 
1) I am checking for both fill authoring patterns. <em><strong> or <strong><em>
2) Your example should show as expected now w/ out those removals. .. Here's why and the fix - We've traditionally trimmed all the button DOM elements that are used to target them. <em><strong> etc.  So in the view it's just an <a> tag (we occasionally style this action-area wrapper) When we start to author w/ these different potential orders and wrappings it can make that button .parentElement scope differ. Since the `.full` is two deep, and sometime authoring will stuff a side by side button example in a wrapping element and only one of them is in another it can get a bit tricky to strip it clean. I have included some logic to target the different button authoring potentials and clean the related DOM (conditional replaceWith) anyhow. [Take a peek](https://rparrish-button-fill--milo--adobecom.hlx.page/drafts/rparrish/links-strong-italics), when you get a sec. Thanks

Reply:
Looking good now!